Reword contribution-related articles to make them easier to read. Also, remove odd escaping in list enumerations (#3990)
ppy/osu-wiki: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/en.md

# GitHub Desktop

> *This article continues from [the main page](/wiki/owcg)* and assumes that you will be using [GitHub Desktop](https://desktop.github.com). **You may use other git clients on your own terms.** Even if you do work locally, you will still need to access GitHub to create pull requests to make your changes happen.

Reviewer: ```suggestion
*This article continues from [the main page](/wiki/osu!_wiki_Contribution_Guide)* and assumes that you will be using [GitHub Desktop](https://desktop.github.com). **You may use other git clients on your own terms.** Even if you do work locally, you will still need to access GitHub to create pull requests to make your changes happen.
```

2. When installation is completed, GitHub Desktop will prompt you to sign in, click `Sign in to GitHub.com`.
3. Fill in the prompts as they appear.
4. Continue to [Cloning](#cloning).

## Cloning

1. Click on `Clone a repository`.

2. This will show a list of your repos; select the `osu-wiki` repo.

3. Select the clone location. It doesn't matter where you clone the repo to, just remember where you cloned it to.

   ![](img/github-desktop-select-repo.jpg)

4. Click `Clone`. This may take a while depending on your network environment.

5. Continue to [Branching](#branching).

## Branching

While branching is technically optional, it is [highly recommended for a couple of reasons](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). To name a few:

- Branches isolate different logical changes from each other: you can work on more than one article and have them reviewed and merged separately.
- When your changes are outside of the `master` branch, you can [safely sync it](/wiki/owcg/Common_Issues#my-branch-is-out-of-date!) without merge conflicts.

1. Click on the `Current branch` dropdown. This is located at the top of the menu.

> 2. Enter a name for this branch. It is suggested to use the name of the article you are editing as the branch name followed by a number. For example, editing the `Forums` article may result in the branch being called `forums1`.

Reviewer: ```suggestion
2. Enter the name of the branch. It is suggested to use the summary of your changes as the branch name. For example, editing the `Forums` article may result in the branch being called `update-forums`.
```
